docs/dev/testing.rst

```diff
@@ -261,3 +262,45 @@ database containing all years of FERC 1 data (equivalent to ``tox -e ferc1_schem
 .. code-block:: console
 
   $ pytest --etl-settings test/settings/full-integration-test.yml test/integration/etl_test.py::test_ferc1_schema
+
+-------------------------------------------------------------------------------
+Data Validation
+-------------------------------------------------------------------------------
+Once the PUDL ETL pipeline has processed all of the available data, we have a
+collection of tests that can be run to verify a collection of expectations
+about what the outputs should look like. These data validation tests are
+organized into datasource specific modules under ``test/validate``. Running
+the full data validation can take as much as an hour, depending on your
+computer.
```

Comment:
Can you run individual tests?

Reply:
Yes, using pytest, just like in the examples above